apps/daffio/src/app/core/footer/docs-footer/docs-footer.component.html

```diff
@@ -1,15 +1,15 @@
 <daff-container size="lg">
   <div class="daffio-docs-footer__grid">
     <div class="daffio-docs-footer__links">
-      @for (link of links; track link) {
+      @for (let link of links) {
```

Comment:
This isn't syntactically correct. Nor does it use the track function. The original was correct.